src/mapping/sanity/reference-client.tsx

```diff
@@ -1,4 +1,30 @@
-// fetch data with graphql query from sanity
-// fetch component and pass data to the component
-// remove data source from component
-export {}
+import { ReferenceClients } from '@src/components/features/decoupled-components/reference-clients/reference-clients';
+import {
+  ReferenceClient,
+  ReferenceClientsProps,
+} from '@src/components/features/decoupled-components/reference-clients/reference-clients.typings';
+import { TeaserItemSanity } from '@src/mapping/sanity/solution';
+
+interface ReferenceClientProps {
```

Comment:
Put typings in separate file and group them where they belong, either in de component folder or the data folder. Mapping should only link existing structures, not define them.